fenix/app/src/main/java/org/mozilla/fenix/search/toolbar/ToolbarView.kt

```diff
@@ -99,10 +102,22 @@ class ToolbarView(
                     R.color.suggestion_highlight_color,
                 ),
                 clear = context.getColorFromAttr(R.attr.textPrimary),
+                pageActionSeparator = ContextCompat.getColor(
```

Comment:
```suggestion
                pageActionSeparator = context.getColorFromAttr(R.attr.borderToolbarDivider),
```
`context.getColorFromAttr` is an extension function that does the same thing here, but simplifies the need to pass in `context` twice. I think we should use this instead to align with how we are already doing this.